components/eam/src/chemistry/modal_aero/modal_aero_convproc.F90

```diff
@@ -2007,6 +2110,10 @@ subroutine compute_wetdep_tend(                              &
    real(r8)     :: expcdtm1     ! exp(cdt) - 1 [unitless]
    real(r8)     :: half_cld     ! 0.5 * cldfrac [fraction]
 
+   real(r8), parameter :: clw_cut = 1.0e-6      ! cutoff value of cloud water for doing updraft [kg/kg]
```

Comment:
add `_r8`

Reply:
`clw_cut` is defined twice in two separate subroutines. Can we make it a module variable as it is parameter?

Reply:
moved it into module parameter